service/openapi.yaml

```diff
@@ -202,6 +202,13 @@ components:
       properties:
         context:
           $ref: '#/components/schemas/context'
+    serverEvaluationSuccess:
+      allOf:
+        - $ref: "#/components/schemas/evaluationSuccess"
+        - properties:
+            cacheable:
+              type: boolean
+              description: Let the provider know that this flag evaluation can be cached
```

Comment:
I doubt whether we should include this in the evaluation response. I think we should only rely on the configuration feature for caching and only cache in provider if `cache. enabled` is set to `true`.

Reply:
The reason why I've added this flag is because not all flags should be cached the same way (because some of them are time-based).

>We don't want to cache all evaluations because some of them contain time-based changes and in those cases, we accept calling multiple times the API because the evaluation result can change and the cache will not be reliable.
By setting the cacheable field we let the decision to the flag management system.

Reply:
> not all flags should be cached the same way

Yeah that's a fair argument. In flagd we avoid caching if the evaluation is based on a targeting match. I will resolve this conversion :)